### PR TITLE
Fixed status command (echo) in isRepo

### PR DESCRIPTION
**Pull request by valinet**
The status reported by `isRepo` fails to print a new line when the directory is not a git repo, because the function returns without printing the conclusion of the checks in one of the possible situations. This fixes this small visual bug. I don't know if the printed messages is the most suitable, but it now includes a new line.

### CODE REVIEW

**Comment by 4s3ti** [comment]
Please chanage the destination branch to test, as per our guidelines: 

https://github.com/pivpn/pivpn/blob/master/CONTRIBUTING.md#do-you-want-to-open-a-pull-request-to-modify-the-source

> Open a new GitHub pull request with the patch on the test branch.

**Comment by orazioedoardo** [comment]
Why is this commit here though? https://github.com/pivpn/pivpn/pull/1194/commits/fc6e853049c24ab490bdaebaaa27a57b100f1c6d

File: auto_install/install.sh [hunk 1]
```diff
@@ -991,7 +991,7 @@ chooseUser(){
 isRepo(){
 	# If the directory does not have a .git folder it is not a repo
 	echo -n ":::    Checking $1 is a repo..."
-	cd "${1}" &> /dev/null || return 1
+	cd "${1}" &> /dev/null || { echo " not found!"; return 1 }
```
Comment (orazioedoardo) [inline auto_install/install.sh:994]:
```suggestion
	cd "${1}" &> /dev/null || { echo " not found!"; return 1; }
```

**Comment by 4s3ti** [comment]
PR inactive for quite long time, implemented this here: https://github.com/pivpn/pivpn/commit/586c631b9ecaf57a1557247e720e6108d7fe0d1e

**Pull request closed without merging**